router_restarter.rb

```diff
@@ -8,6 +8,8 @@
 def restart_router
   StateManager::update_state("Offline, Restarting.")
   RouterAutomation.do_restart
+  RouterAutomation.do_restart
+  p "SSD"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.